brownfield_django/settings_shared.py

```diff
@@ -37,6 +37,7 @@
     'pagetree',
     'pageblocks',
     'quizblock',
+    'storages',
```

Comment:
it doesn't really matter and doesn't hurt anything, but `storages` actually doesn't need to be in `INSTALLED_APPS` (at least for S3 stuff)